### ISSUE TITLE
Some questions about paper details

### ISSUE THREAD

**Issue by FightForCS** [lawy623/LSTM_Pose_Machines#3]
Hi, thanks for your great paper and code. I have some questions: 

1. How long does it take to pretrain on LEEDS Sport Dataset and MPII Dataset ? And How long does it take to train on PENN dataset after pretraining?

2. I notice the network does not use feature extractor which is pretrained on ImageNet. Why not? Since it might speed up training, as CPM author stated  in https://github.com/shihenw/convolutional-pose-machines-release/issues/38

3. How about the performance if we only train on PENN dataset, without pretraining on LEEDS Sport Dataset and MPII Dataset? Will the performance drop a lot ?

Thanks.

**Comment by pasawaya** [comment]
Not one of the authors but I wrote a Pytorch implementation and got up to 86% accuracy in 24 hours when pre-training on MPII.

**Comment by YHDang** [comment]
> Not one of the authors but I wrote a Pytorch implementation and got up to 86% PCK@0.2 accuracy in 24 hours when pre-training on MPII.

@pasawaya If is it convenient for you to release the Pytorch implementation, please? I have met some problems during reproducing the data preprocessing . Thanks
